frontend/src/customAxios.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import { notification } from "antd";
+import * as String from "@/constants/strings";
+
+const UNAUTHORIZED = 401;
+const MAINTENANCE = 503;
+
+const CustomAxios = (errorToastMessage) => {
+  const instance = axios.create();
```

Comment:
Any reason why we wouldn't set the `baseURL` here so that we can remove `ENVIRONMENT.apiUrl` from every API call? And same question about request headers.

Everything else here looks good to me

Reply:
Ticket scope. There is a good argument to be made to make all these changes at once but the scope of work was only set to cover the response interceptors.